File: URI-lua/1041.lua
x,y = io.read("*n","*n")
if x==0 and y==0 then
  io.write("Origem\n")
elseif x == 0 then
  io.write("Eixo Y\n")
elseif y == 0 then
  io.write("Eixo X\n")
elseif x > 0 then
  if y > 0 then
    io.write("Q1\n")
  else
    io.write("Q4\n")
  end
else
  if y > 0 then
    io.write("Q2\n")
  else
    io.write("Q3\n")
  end
end

File: URI-c#/1041.cs
using System;

class Program
{
    static void Main()
    {
        string saida;
        string[] values = Console.ReadLine().Split(" ");
        double x = Convert.ToDouble(values[0]), y = Convert.ToDouble(values[1]);
        if (x == 0 && y != 0)
        {
            saida = "Eixo Y";
        }
        else if (x != 0 && y == 0)
        {
            saida = "Eixo X";
        }
        else if (x == 0 && y == 0)
        {
            saida = "Origem";
        }
        else if (x > 0 && y > 0)
        {
            saida = "Q1";
        }
        else if (x > 0 && y < 0)
        {
            saida = "Q4";
        }
        else if (x < 0 && y > 0)
        {
            saida = "Q2";
        }
        else
        {
            saida = "Q3";
        }
        Console.WriteLine(saida);
    }
}